lib/util/lifecycleMethods.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @fileoverview lifecycle methods
+ * @author Tan Nguyen
+ */
+
+'use strict';
+
+module.exports = [
+  'getDefaultProps',
+  'getInitialState',
+  'getChildContext',
```

Comment:
some of these are additions that will affect `no-typos` also. could we add some tests that cover those?

Reply:
@ljharb it already updated, please lemme know if anything needs to be updated